src/content/caseStudies/contentful-share-pear/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Content for Everyone"
+description: "Building Public-Sharing of Blog Drafts in 1 Month"
+publishDate: "June 20, 2023"
+authors:
+  - astro-team
+lang: "en"
+headerImage: /src/content/blog/_images/contentful-share-pear/header.webp
+---
+
+import shareFeature from "./share_feature.webp"
+import results from "./results.webp"
+import BlogContentImage from "~/components/BlogContentImage.astro"
+import ImageFigure from "~/components/ImageFigure.astro"
+import Blockquote from "~/components/Blockquote.astro"
+
+<div class="text-white space-y-8">
+	<div class="text-2xl leading-normal tracking-normal text-astro-gray-100">
```

Comment:
Not sure what the best way to accomplish this styling is, so I just hardcoded it in the content for now 😅